build/launch_training.py

```diff
@@ -67,69 +66,31 @@ def main():
     logging.basicConfig(level=LOGLEVEL)
 
     logging.info("Attempting to launch training script")
-    parser = transformers.HfArgumentParser(
-        dataclass_types=(
-            configs.ModelArguments,
-            configs.DataArguments,
-            configs.TrainingArguments,
-            peft_config.LoraConfig,
-            peft_config.PromptTuningConfig,
-        )
-    )
-    peft_method_parsed = "pt"
+
     json_path = os.getenv("SFT_TRAINER_CONFIG_JSON_PATH")
     json_env_var = os.getenv("SFT_TRAINER_CONFIG_JSON_ENV_VAR")
 
     # accepts either path to JSON file or encoded string config
     if json_path:
-        (
-            model_args,
-            data_args,
-            training_args,
-            lora_config,
-            prompt_tuning_config,
-        ) = parser.parse_json_file(json_path, allow_extra_keys=True)
-
-        contents = ""
         with open(json_path, "r", encoding="utf-8") as f:
-            contents = json.load(f)
-        peft_method_parsed = contents.get("peft_method")
-        logging.debug("Input params parsed: %s", contents)
+            job_config_dict = json.load(f)
     elif json_env_var:
         job_config_dict = txt_to_obj(json_env_var)
```

Comment:
Note to self: perhaps we should refactor this into build/utils as well since accelerate_launch and launch_training use the same method to parse the JSON